### PR TITLE
vignette on meta_forestly

### PR DESCRIPTION
**Pull request by yuliasidi**
To clarify the addition of different parameters, including completely new parameters (dose reduction example)

### CODE REVIEW

File: vignettes/meta_forestly.Rmd
Comment (nanxstats) [inline vignettes/meta_forestly.Rmd:1]: You might want to rename it to `meta-forestly.Rmd` to be consistent with the existing naming scheme.

File: vignettes/meta_forestly.Rmd [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+```
+
+```{r}
+library(forestly)
+library(metalite)
+```
+
+# Overview
+
+`meta_forestly()` is the first step in the Interactive Forest Plot generation workflow and is a wrapper of several [metalite](https://merck.github.io/metalite/) functions. Specifically, it defines a metadata, including adam datesets, population, observations and parameters to be used in the forest plot.
```
Comment (nanxstats) [inline vignettes/meta_forestly.Rmd:32]: - Why is "Interactive Forest Plot" in title case? If it's not a title, use regular case.
- Remove the link to pkgdown site.
- "metadata" is not a countable noun.
- "adam" should be stylized as "ADaM".

File: vignettes/meta_forestly.Rmd [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+```{r}
+library(forestly)
+library(metalite)
+```
+
+# Overview
+
+`meta_forestly()` is the first step in the Interactive Forest Plot generation workflow and is a wrapper of several [metalite](https://merck.github.io/metalite/) functions. Specifically, it defines a metadata, including adam datesets, population, observations and parameters to be used in the forest plot. 
+In this [vignette](https://merck.github.io/forestly/articles/forestly.html#define-metadata) we will take a closer look at the `parameter_term` argument in `meta_forestly()` to better understand how to generate the forest plot for different parameters of interest.
```
Comment (nanxstats) [inline vignettes/meta_forestly.Rmd:33]: "In this vignette" points to another vignette - this is confusing. Maybe you don't need a link at all.

File: vignettes/meta_forestly.Rmd [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+# `parameter_term` examples
+
+Prior to turning to the examples, just a reminder that the treatment group variable in the ADaM datasets should be set as a factor, otherwise a warning will be generated. 
+For example, if this is what you have in the `ADSL.TRT01A`:
```
Comment (nanxstats) [inline vignettes/meta_forestly.Rmd:38]: It's unclear what the `ADSL.TRT01A` format mean. The pandas way to mention a variable in a data frame?

File: vignettes/meta_forestly.Rmd [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+```{r}
+forestly_adae$TRTA <- factor(
+  forestly_adae$TRTA, 
+  levels = c("Xanomeline Low Dose", "Placebo"), 
+  labels = c("Low Dose", "Placebo")
+  )
+```
+
+If the deliverable requires a forest plot of all adverse events and serious adverse events, then the following code could be used with`any` and `ser` parameters in `parameter_term` argument respectively:
```
Comment (nanxstats) [inline vignettes/meta_forestly.Rmd:63]: Need a whitespace before `any`

File: vignettes/meta_forestly.Rmd [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+Now, similar to the originally generated forest plot above, we need to apply `prepare_ae_forestly()`, `format_ae_forestly()`, and `ae_forestly()` to generate the plot that will include adverse events which resulted in dose reductions:
+
+```{r}
+meta_new |>
+  prepare_ae_forestly() |>
+  format_ae_forestly() |>
+  ae_forestly()
+```
+
+
```
Comment (nanxstats) [inline vignettes/meta_forestly.Rmd:145]: Only need 1 (one) new line at the end of file.

**Review by nanxstats — CHANGES_REQUESTED** [review]

File: vignettes/meta_forestly.Rmd (hunk 1 above)
Comment (yuliasidi) [reply vignettes/meta_forestly.Rmd]: why not to keep the link to metalite pkgdown site?

File: vignettes/meta_forestly.Rmd (hunk 3 above)
Comment (yuliasidi) [reply vignettes/meta_forestly.Rmd]: it is how you would expect it to appear in the spec, but I can remove ADSL part. It should be clear without it too

File: vignettes/meta_forestly.Rmd (hunk 1 above)
Comment (nanxstats) [reply vignettes/meta_forestly.Rmd]: It's ok to keep it.

**Comment by nanxstats** [comment]
To make this pass `R CMD build`, simply attach the datasets before using them in the vignette:

```r
data(forestly_adsl)
data(forestly_adae)
```

To avoid the `R CMD check` note saying "sub-directories of 1Mb or more", add this line to `.Rbuildignore`:

```
^vignettes/meta-forestly.Rmd$
```

File: vignettes/.gitignore [hunk 6]
```diff
@@ -1,2 +1,3 @@
 *.html
 *.R
+^vignettes/meta-forestly.Rmd$
```
Comment (nanxstats) [inline vignettes/.gitignore:3]: This should be added to `.Rbuildignore`, not `.gitignore`.

**Review by nanxstats — CHANGES_REQUESTED** [review]

File: vignettes/.gitignore (hunk 6 above)
Comment (yuliasidi) [reply vignettes/.gitignore]: done

**Review by nanxstats — APPROVED** [review]

**Comment by nanxstats** [comment]
It's up: https://merck.github.io/forestly/articles/meta-forestly.html

**Pull request merged**